### PR TITLE
Test and import game fix

### PR DESCRIPTION
**Pull request by inche-ali**
Fix for test script related with PR #232  - updated some testcase, test_schema, test data
Fix for game import game - missing input element in AdminController 
                                         - add checking using "instanceof KeyedContainer"

### CODE REVIEW

File: src/models/Control.php [hunk 1]
```diff
@@ -119,24 +119,24 @@ class Control extends Model {
   }
 
   public static async function importGame(): Awaitable<bool> {
-    $data_game = self::readJSON('game_file');
-    if ($data_game) {
-      $logos = must_have_idx($data_game, 'logos');
+    $data_game = JSONImporterController::readJSON('game_file');
+    if ($data_game instanceof KeyedContainer) {
```
Comment (gsingh93) [inline src/models/Control.php]: If the data isn't a KeyedContainer we should return `false` early. You can also `error_log` what the error was so we can find it in the logs later.

File: src/models/Control.php [hunk 2]
```diff
@@ -119,24 +119,24 @@ class Control extends Model {
...
+      $logos = array_pop(must_have_idx($data_game, 'logos'));
       $logos_result = await Logo::importAll($logos);
```
Comment (gsingh93) [inline src/models/Control.php]: Does this pass typechecking (`hh`)? The documentation says `array_pop` can return null, but `Logo::importAll` can't take null.

**Comment by inche-ali** [comment]
Thanks for reviewing my fix. I will try to modify it

**Comment by inche-ali** [comment]
OK..I fixed it.
please review

#hacktoberfest :)

**Comment by gsingh93** [comment]
Looks good to me. Just to confirm, you did run `hh_client` on it this time, right? Also, run `extra/format.sh` in case there are any formatting changes that need to be made (make sure to rebase first).

**Comment by inche-ali** [comment]
ahh.. thanks for reminding me, forgot that to run that format.sh & hh_client.. 
I've pushed another changes, after fix some type checking.. should be okay now..

#hacktoberfest :)

**Comment by inche-ali** [comment]
done rebasing..

**Comment by gsingh93** [comment]
Thanks!

**Pull request merged**